recipe/meta.yaml

```diff
@@ -14,25 +14,24 @@ source:
   sha256: 71ee0e0a25d6acc5e99ff13c287cae006604f6054dd9b60c352d0216e5d6b6f4
 
 build:
-  skip: True  # [py<=36]
+  skip: True  # [py<38]
   number: {{ build }}
 
 requirements:
   build:
     - {{ compiler("c") }}
   host:
     - python
-    - cython >=0.29.15
-    - hdf5
-    - numpy
+    - cython >=0.29.15,<3
+    - hdf5 1.12.1
```

Comment:
if the goal is to use hdf5 1.12.1 everywhere would not be better to add hdf to aggregate/conda_build_config.yaml to separate aggregate branch ?

Reply:
I sincerely think that CBC would be bad here. It's there right now, but there is two versions (one for osx-arm64 and one for the others) and I'll sincerely say that all this worked was caused by CBC having different versions.

Generally, when I speak about consistency, I mean consistency between variants/platforms for a version X. For example, if you use h5py on mac and you get hdf5-1.12.1 and on linux you get 1.10.1, this is inconsistent and quite problematic. When I use a version of a package, I expect it to have the same set of dependencies across platforms. Even more so when there is a file format (or similar things like hdf5) involved.